tds/routers/datasets.py

```diff
@@ -95,8 +104,8 @@ def update_feature(
     return "Updated Feature"
 
 
-@router.delete("/features/{id}")
-def delete_feature(id: int, rdb: Engine = Depends(request_rdb)) -> str:
+@router.delete("/features/{id}", **update.fastapi_endpoint_config)
```

Comment:
Should this be the delete config instead of the update?

Reply:
good catch